ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -766,6 +770,7 @@ def generate_ping_dut_lo(self):
                                     ip_dst=dut_lo_ipv4)
 
         self.ping_dut_exp_packet  = Mask(exp_packet)
+        self.ping_dut_exp_packet.set_do_not_care_scapy(scapy.Ether, "src")
```

Comment:
I think you don't need this, once you remove the change for src_mac from above.

Reply:
Removed